server/routes/categories.js

```diff
@@ -0,0 +1,26 @@
+const router = require('express').Router()
+const Category = require('../database/models/category.js')
+
+router.get('/', (req, res, next) => {
+  Category.findAll()
+    .then(categories => {
+      res.json(categories)
+    })
+    .catch(e => {
+      console.error(e)
+      next(e)
+    })
+})
+
+router.get('/:id', (req, res, next) => {
+  Category.findbyId(req.params.id)
```

Comment:
Is this working? `.findByPk()` ?

Reply:
Don't see a change here, needs to be `Category.findByPk(req.params.id)`